tests/src/components/Toaster/ToasterTest.tsx

```diff
@@ -0,0 +1,64 @@
+import { onMount, onCleanup } from 'solid-js';
+import useToast from '@components//Feedback/Toast/toast';
+import Tab from '@components/Layout/Tab/Tab';
+import './ToastTest.css';
+
+interface ToastEventDetail {
+    position?:
+        | 'top-left'
+        | 'top-right'
+        | 'bottom-left'
+        | 'bottom-right'
+        | 'top-center'
+        | 'bottom-center'
+        | 'middle-left'
+        | 'middle-right'
+        | 'middle-center';
+    text?: string;
+    timeout?: number;
+}
+
+const ToasterTest = () => {
+    const [Toaster, createToast] = useToast();
+
+    const setupToast = (event: Event) => {
+        const { detail } = event as CustomEvent<ToastEventDetail>;
+        createToast({
+            position: detail.position || 'top-right',
+            body: (close, progress) => {
+                return (
+                    <div class="gf-toast">
+                        <div class="gf-toast-text">{detail.text || 'Default Toast'}</div>
+                        {close(<button class="gf-toast-close-button">X</button>)}
+                        <div
+                            class="gf-toast-progress"

```

Comment:
If you add a selectors json you can use it like so:
```suggestion
                    <div class={selectors.toast}>
                        <div class={selectors.toastTest}>{detail.text || 'Default Toast'}</div>
                        {close(<button class={selectors.toastCloseButton}>X</button>)}
                        <div
                            class={selectors.toastProgress}
```
That way you don't have to hardcode the classes in both test and spec pages